### ISSUE TITLE
Add link to TwinPack registration page

### ISSUE THREAD

**Issue by bengeisler** [Zeugwerk/Twinpack#93]
First, thanks for creating this package manager! I think it's really useful.

However, I have a small request: 
When first publishing a package to TwinPack you can register as a new user and create a TwinPack account. However, apart from that link within the publishing page in the TwinPack extension, there is no link to that account management page available. It would be useful to include that link in the readme, so that users can e.g. change their password. 

TLDR: My password manager didn't save the password and I need a way to reset it :)

**Comment by iadonkey** [comment]
Thanks for you feedback, we will add one

in the meantime you can reset the password here https://zeugwerk.dev/wp-login.php?action=lostpassword
